src/components/PostList/PostList.jsx

```diff
@@ -1,3 +1,23 @@
-export const PostList = () => (
-  <>Put the list here</>
+import { PostInfo } from '../PostInfo';
+
+export const PostList = ({ posts, comments, users }) => (
+  <div className="PostList">
+    {posts.map((post) => {
+      const postUser = users.find(user => user.id === post.userId);
+      const postComments = comments.filter(
+        comment => comment.postId === post.id,
+      );
```

Comment:
this operations will be executed during eacn render. better to do it in App.jsx outside of component function and work with extended posts (with user and comments fields)